conan/tests/cocpp19/conanfile_wrong.txt

```diff
@@ -0,0 +1,8 @@
+[requires]
```

Comment:
this file seems wrong by name. Is it required?

Reply:
I delete this file